packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -759,10 +792,58 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
     )
   }
 
-  private handleSelectedItemRemove(e: React.SyntheticEvent, item: ShorthandValue) {
+  private handleSelectedItemKeyDown(
+    e: React.SyntheticEvent,
+    item: ShorthandValue,
+    predefinedProps: DropdownSelectedItemProps,
+    DropdownSelectedItemProps: DropdownSelectedItemProps,
+  ) {
+    const { activeIndex, value } = this.state as { activeIndex: number; value: ShorthandValue[] }
+    switch (keyboardKey.getCode(e)) {
+      case keyboardKey.Delete:
+      case keyboardKey.Backspace:
+        this.handleSelectedItemRemove(e, item, predefinedProps, DropdownSelectedItemProps)
+        break
+      case keyboardKey.ArrowLeft:
```

Comment:
In RTL mode, left/right arrow keys need to be swapped. (applies to line 814 as well)